base/commands/shell_script_common.go

```diff
@@ -48,7 +47,7 @@ func makeEndLineFunc() shell.EndLineFn {
 	}
 }
 
-func makeTextFunc(m *cmd.Main, ec plug.ExecContext, verbose, ignoreErrors, echo bool, sf shortcutFunc) shell.TextFn {
+func makeTextFunc(m *cmd.Main, ec plug.ExecContext, verbose bool, sf shortcutFunc) shell.TextFn {
```

Comment:
Let's keep the signature as is even though `ie` and `echo` are not used at the moment.